roles/openshift-patch-resource/README.md

```diff
@@ -0,0 +1,41 @@
+openshift-patch-resource
+========================
+
+Patch a Kubernetes/Openshift resource
+
+Requirements
+------------
+
+This role requires the `kubernetes` python module
```

Comment:
Should we also mentioned a "valid" or "active" session to an OpenShift (or k8s) cluster as a requirement?